src/Raven.Studio/wwwroot/App/views/database/tasks/editOlapEtlTask.html

```diff
@@ -221,81 +221,81 @@ <h3 class="margin-top margin-top-sm">The default ID column name is <code>_id</co
         </div>
         <div class="col-xs-12 col-lg-6 flex-vertical" data-bind="with: editedTransformationScriptSandbox()">
             <div class="flex-grow">
-                <div class="panel" data-bind="if: $root.showEditTransformationArea">
-                    <div class="padding padding-sm">
-                        <div data-bind="visible: !isNew()">
-                            <label>
-                                <strong>Name:</strong> <span data-bind="text: name"></span>
-                            </label>
+                <div class="panel padding" data-bind="if: $root.showEditTransformationArea">
+                    <div class="flex-horizontal margin-bottom" data-bind="validationElement: name">
+                        <label class="control-label"><strong>Name:</strong></label>
+                        <div class="flex-grow margin-left">
+                            <input type="text" class="form-control" placeholder="Enter script name (optional)" autocomplete="off" 
```

Comment:
name isn't optional - we just autogenerate them.

Reply:
done - showing "Script_1" immediately - upon opening